test/runtests.jl

```diff
@@ -90,12 +90,16 @@ else
     @test start(t) == nothing
     @test length(read(t, UInt8, 6)) == 12
     @test stop(t) == nothing
-    @test NIDAQ.CfgSampClkTiming(t.th, convert(Ref{UInt8},b""), 100.0, NIDAQ.Val_Rising,
+    rslt = Ref(UInt32(0))
```

Comment:
Ref{UIn32}(0) is the preferred syntax i believe.  see https://docs.julialang.org/en/latest/manual/calling-c-and-fortran-code/#Passing-Pointers-for-Modifying-Inputs-1